synapse/cortex.py

```diff
@@ -2313,6 +2397,9 @@ async def loadStormPkg(self, pkgdef):
         for cdef in cmds:
             await self._setStormCmd(cdef)
 
+        for gdef in pkgdef.get('graphs', ()):
+            await self._addStormGraph(gdef)
```

Comment:
This persists the graph data, which violates the docstring for this function ( which is not to persist the data loaded ).

Reply:
Ah yep, i'll fix that